src/notifications/tasks.py

```diff
@@ -28,8 +30,29 @@ def email_notifications_to_users(root_url):
     # send_email_notification_tenant.delay(root_url)
 
 
+def get_users_to_email_for_notifications():
+    students_to_email = list(
```

Comment:
These to lists of emails seem almost the same as the lists in the other method.  These should come from a single common method rather than repeating ourself, and then apply any extra filters and flattening afterwards.

Instead of duplicating code, can you put these into a method, perhaps with a couple parameters to indicate students, staff, etc?  That would prevent the quadruplication of this code which we are repeating 4 times:

```
.filter(emailaddress__verified=True, emailaddress__primary=True)
                    .exclude(email='')
                    .exclude(email__isnull=True)
```
Actually, why do the two teacher lists exclude email__is_null=True but the two student lists do not?

Reply:
Refactored the code and made it to the `Profile.objects.get_mailing_list`.. that method wasn't used anywhere so I thought that was a good place to put it